.eslintignore

```diff
@@ -1,4 +1,3 @@
 bower_components/**
 node_modules/**
 tests/node_modules/**
-src/bloomdata_short_pwd.js
```

Comment:
You can leave this in. On the off chance somebody is using `eslint` to lint things and not going through the Gulp task.

On that note, not sure if we need "tests/node_modules/**" entry